examples/udp-connect-client/udp_connect_client.bal

```diff
@@ -0,0 +1,53 @@
+// This is the connection oriented client implementation for the UDP socket.
+import ballerina/io;
+import ballerina/udp;
+
+public function main() returns error? {
+  
+    // Create a new connection-oriented UDP client by providing the 
+    // `remoteHost` and the `remotePort`.
+    // Optionally, you can provide the interface that the socket needs to bind 
+    // and the timeout in milliseconds, which specifies the read timeout value.
+    // udp:Client client = new ("www.ballerina.com", 80,
+    //                         localHost = "localhost", timeoutInMillis = 2000);
+    udp:ConnectClient socketClient = check new("www.ballerina.com", 80);
```

Comment:
"www.ballerina.com" has a UDP server at port 80? Let's make sure the sample executes as it says.

Reply:
updated the example to `check new("localhost", 48829)` which is used to write udp:LIstener example